examples/ethereum-cryptopunks-aggregations/src/crypto-punks.ts

```diff
@@ -0,0 +1,16 @@
+import { PunkBought as PunkBoughtEvent } from '../generated/CryptoPunks/CryptoPunks';
+import { PunkBought } from '../generated/schema';
+
+export function handlePunkBought(event: PunkBoughtEvent): void {
+  let entity = new PunkBought('dummy-id');
```

Comment:
For clarity, use `auto`; for any entity where the idtype is `Int8` or `Bytes`, you can set the `id` to `auto` and `graph-node` will make up a unique id. But we should really change `graph-cli` so that for time series, the generated code has a constructor that takes no arguments.